tests/test_gruptree.py

```diff
@@ -435,14 +435,6 @@ def test_emptytree_commandlinetool(tmp_path, mocker, caplog):
     assert "No tree data to prettyprint" in caplog.text
 
 
-def test_cli_nothing_to_do(mocker, capsys):
```

Comment:
Removal of this test indicates some change in behavior, in this case "when DATA is supplied, but no action". Do we know if this is acceptable (or disruptive) for current usage?

Reply:
All it used to do was check if sys.exit is called if there are no results to produce, but one could off course bring it back, but change the test..